packages/api/endpoints/collections.js

```diff
@@ -43,14 +44,22 @@ const log = new Logger({ sender: '@cumulus/api/collections' });
  * @returns {Promise<Object>} the promise of express response object
  */
 async function list(req, res) {
+  log.trace(`list query ${JSON.stringify(req.query)}`);
```

Comment:
Nit: we almost never use trace over .info.        A decision to start doing that should be deliberate

Reply:
Ok, sure, that comment applies to both PRs then - this is in theory a change to how we're logging.   We should broadcast (arch meting)  and decide to do it, else efforts outside this insular work won't be consistent with the approach you're choosing. 

It's minor and shouldn't block the PR (nit), but it's notable, particularly if we ever get around to log-level squelching. 

@jennyhliu

Reply:
I will add a comment to the feature branch, so we can revisit when Jenny is back and when we are thinking of merging the feature branch.